src/generator/writers/writeStdlib.ts

```diff
@@ -1161,7 +1161,8 @@ function genTactDictGet(
     value: ValType,
 ): void {
     const signatureKeyType = key === "uint" ? "int" : key;
-    const signatureRetType = value === "uint" ? "int" : value;
+    const signatureRetType =
```

Comment:
Looks like this should be refactored into a helper function (I should have done it in my previous refactoring)